lib/form-validation.js

```diff
@@ -0,0 +1,6708 @@
+(function (global, factory) {
```

Comment:
graag in een comment vermelden wat je net hebt aangepast want das nogal moeilijk te weten anders

Reply:
Hierbij een screenshot van de PR bij AIV, dezelfde code werd toegepast in het dist bestand zodat we de feature al vroeger hebben, hier werd gebruik gemaakt van ES5.

<img width="2880" alt="PR" src="https://user-images.githubusercontent.com/11139249/78970943-c13b4d80-7b0a-11ea-9718-33bb7c92824e.png">

Reply:
de code verwart mij een beetje dat daar jaar + dag + maand + ... staat en dat dat strings zijn en geen nummers. maar ze worden eerst gematcht om dan gewoon terug aan mekaar te plakken. dus ik zou eerder eerst het patroon checken en dan splitten in "laatste 2 karakters" en "alles behalve laatste 2 karakters"

Reply:
Probleem is dat er . en - tekens tussen staan, dus dit lijkt me eleganter dan die er uit te strippen, nu gebruik ik de groups uit de regex. In ieder geval, PR staat open bij AIV. Ik zal afwachten wat hun feedback is op de code as is, het is hun bron uiteindelijk.

Reply:
Ja, ik zou wel strippen maar ca va.